api/v1/routes/waitlist.py

```diff
@@ -95,4 +96,30 @@ def admin_add_user_to_waitlist(
         data = {"email": item.email, "full_name": item.full_name},
     )
 
-    return JsonResponseDict(**resp)
\ No newline at end of file
+    return JsonResponseDict(**resp)
+
+@waitlist.get("/users", response_model=success_response, status_code=200)
+async def get_all_waitlist_emails(
+    request: Request,
+    db: Session = Depends(get_db),
+    admin=Depends(get_super_admin)
+):
+    try:

```

Comment:
No need for the try and except block. The app exception handler is handling it

Reply:
I've resolved this